### PR TITLE
Localization changes and improvements

### PR DESCRIPTION
**Pull request by Satton2**
• Russian localization improvements.
• I cannot test the addon properly (_no default classes_), but I guess "Reset" fits more here.
• Do you really need all these words to be started from capitals?

### CODE REVIEW

File: lua/terrortown/lang/deutsch/tttc.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-local L = LANG.GetLanguageTableReference("deutsch")
+local L = LANG.GetLanguageTableReference("Deutsch")
```
Comment (TimGoll) [inline lua/terrortown/lang/deutsch/tttc.lua:1]: why did you change this to a capital letter? While both work, we use lowercase letters for the identifier.

File: lua/terrortown/lang/english/tttc.lua [hunk 2]
```diff
@@ -1,8 +1,8 @@
-local L = LANG.GetLanguageTableReference("english")
+local L = LANG.GetLanguageTableReference("English")
```
Comment (TimGoll) [inline lua/terrortown/lang/english/tttc.lua:1]: same here

File: lua/terrortown/lang/english/tttc.lua [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
-L["ttt2_tttc_abort_ability"] = "Abort Ability Preview"
-L["ttt2_tttc_class_ability"] = "Class Ability"
-L["ttt2_tttc_drop_class"] = "Drop Class"
+L["ttt2_tttc_abort_ability"] = "Abort ability preview"
+L["ttt2_tttc_class_ability"] = "Class ability"
+L["ttt2_tttc_drop_class"] = "Reset class"
```
Comment (TimGoll) [inline lua/terrortown/lang/english/tttc.lua:5]: No, this is something completely different. "Remove Class" would be ok as well.

**Review by TimGoll — CHANGES_REQUESTED** [review]

**Comment by Alf21** [comment]
@Satton2 just to check whether that's still alive

**Comment by Satton2** [comment]
> @Satton2 just to check whether that's still alive

Since I am alive, then this PR also alive. I just don't have time, sorry.

**Pull request closed without merging**